tasks/console.js

```diff
@@ -0,0 +1,196 @@
+'use strict';
+
+var repl = require('repl');
+var fs = require('fs');
+var path = require('path');
+var rewire = require('rewire');
+var async = require('async');
+
+var AppConfig = require('../helpers/config.js');
+var Sequence = require('../helpers/sequence.js');
+var cache = require('../helpers/cache.js');
+var z_schema = require('../helpers/z_schema.js');
+var Logger = require('../logger.js');
+var appConfig = AppConfig(require('../package.json'));
+
+var logger = new Logger({
+	echo: appConfig.consoleLogLevel, errorLevel: appConfig.fileLogLevel,
+	filename: appConfig.logFileName
+});
+var genesisblock = require('../genesisBlock.json');
+var modules = [];
+
+var modulesInit = {
+	accounts: '../modules/accounts.js',
+	blocks: '../modules/blocks.js',
+	dapps: '../modules/dapps.js',
+	delegates: '../modules/delegates.js',
+	loader: '../modules/loader.js',
+	multisignatures: '../modules/multisignatures.js',
+	node: '../modules/node.js',
+	peers: '../modules/peers.js',
+	signatures: '../modules/signatures.js',
+	system: '../modules/system.js',
+	transactions: '../modules/transactions.js',
+	transport: '../modules/transport.js'
+};
+
+async.auto({
```

Comment:
Very similar thing is already done in test/node.js:initApplication. Can the function be reused?

Reply:
I will check

Reply:
Done.